README.md

```diff
@@ -3,4 +3,6 @@ tools that was previously included in https://github.com/giantswarm/mayu
 
 
 is used for listing  machines registered to `mayu`
-```
+
+# Docker image
+Image is avaiable at `quay.io` - https://quay.io/repository/giantswarm/mayuctl
```

Comment:
Why not having the quay badge in the head of each readme? I would favour this. Having badges for circle and quay everywhere would be super nice.